FE/src/pages/QuestionCreationPage/QuestionCreationPage.tsx

```diff
@@ -51,20 +69,25 @@ const QuestionCreationPage = () => {
   };
 
   // 폼 제출 핸들러
-  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
     // 문서에디터 내용 html 형식으로 변환
     const contentState = draftToHtml(
       convertToRaw(editorState.getCurrentContent()),
     );
-    handleInputChange('content', contentState);
-    // formData를 서버로 전송하는 로직 추가할 예정.
+    // 서버로 보낼 데이터 가공
+    const createQuestionData = {
+      ...formData,
+      content: contentState,
+    };
+    const res = await createQuestionAPI(createQuestionData);
+    // 후속 처리할 예정
```

Comment:
지금 이 부분 build 에러 나네요 (사용하지 않는 변수 존재)
현재 draftId도 로직에서 제외되어 있어 API가 제대로 post되지 않으니, 이번 PR에서는 API 실행 로직은 제외하는게 좋을 것 같아요.
다음과 같이 변경하는 것 어떠신가요?
```suggestion
    console.log('서버로 보낼 데이터 :', createQuestionData);
    return alert('현재 구현중인 기능입니다');

    await createQuestionAPI(createQuestionData);
    // 후속 처리할 예정
```

Reply:
제가 build 테스트를 할 때에는 에러가 나지 않았는데 제가 이미 draft 관련 작업을 하면서 제 local 에서는 해당 라인이 다르게 되어있어서 에러가 나지 않았던 것 같네요. 😢